gbp/pkg/pristinetar.py

```diff
@@ -73,15 +73,21 @@ def get_commit(self, archive_regexp):
         @type archive_regexp: C{str}
         """
         if not self.repo.has_pristine_tar_branch():
-            return None
+            return None, False
 
         regex = ('pristine-tar .* %s' % archive_regexp)
         commits = self.repo.grep_log(regex, self.branch, merges=False)
         if commits:
             commit = commits[-1]
             gbp.log.debug("Found pristine-tar commit at '%s'" % commit)
-            return commit
-        return None
+            files = self.repo.get_commit_info(commit)['files']
```

Comment:
maybe move the part that looks for the signature into a new (private) method so it becomes obvious what's going on.